hokusai/templates/docker-compose-service-dev.yml.j2

```diff
@@ -0,0 +1,4 @@
+    image: "hokusai_force:${BUILD_TAG:-latest}"
+    build:
+      context: ../
+      dockerfile: Dockerfile.dev
```

Comment:
I've opted to create a new dockerfile specifically for dev rather than add more stages to the existing Dockerfile, which is already complex. I believe this to be a simpler and saner solution, but I'm happy to hear other opinions.